### ISSUE TITLE
Use the server without email

### ISSUE THREAD

**Issue by AustralEpitech** [ferdium/ferdium-server#71]
I would love to create the users myself for my instance of Ferdium. I didn't see any docs to do so. Is it possible to create user with just `username:password` ? Or better yet, connect Ferdium to an already existing ldap server?

**Comment by thuvh** [comment]
I installed ferdium-server for myself with flag `IS_REGISTRATION_ENABLED` on, then sign up with my email. After that, turn off this flag and reboot server :D

**Comment by AustralEpitech** [comment]
Yeah, I tried that but I believe you need to setup the smtp server to enable this and I didn't want to do it.
Guess I have no choice

**Comment by thuvh** [comment]
yep, can you share the reason why you didn't want to do that? I think it's easy with gmail or st like that.

**Comment by AustralEpitech** [comment]
I just don't like the fact that the server requests uneeded information about my users.
Also, I am starting to self host my own mail server, but it takes a long time and I can't fully commit atm, so my installation is a bit clumsy.

**Comment by deathbybandaid** [comment]
I would also like to authenticate with ldap or radius.

**Comment by santiagon610** [comment]
> I would also like to authenticate with ldap or radius.

OIDC/SAML would be my vote 😀
